aws-csharp/Aws.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Nit, found it odd this was named `Aws.csproj` ... Makes it sound like my project is building an AWS, rather than my project simply using/deploying to it. Does the name actually matter? What is standard convention?

Reply:
The name doesn't matter, we can pick whatever we like. Ideally, we could gen it from the project name, but I haven't checked if that's possible.

Reply:
Renamed to `Infra.csproj` for now. Better ideas are welcome.

Reply:
> is there a way to generate this file using a name derived from the project name?

Not currently. We do already [support replacing content inside files](https://github.com/pulumi/pulumi/blob/105eb210ce7394e73006f677922aa6d705e618ba/pkg/workspace/templates.go#L736-L746), but not the file names themselves.

Adding support for transforming file names would be a relatively simple change.

Reply:
Added to the list in https://github.com/pulumi/pulumi/issues/3406